desc/equilibrium/equilibrium.py

```diff
@@ -1586,7 +1586,15 @@ def resolution_summary(self):
 
     @classmethod
     def from_near_axis(
-        cls, na_eq, r=0.1, L=None, M=8, N=None, ntheta=None, spectral_indexing="ansi"
+        cls,
+        na_eq,
+        r=0.1,
+        L=None,
+        M=8,
+        N=None,
+        ntheta=None,
+        spectral_indexing="ansi",
+        w=2,
```

Comment:
In theory, should this be related to the "order" of the NAE constraints we use? If so, I would recommend changing the variable name to `order` or whatever we use in those NAE constraint functions.

Reply:
I don't think theres a direct relation. While the order of the expansion usually means the distance from the axis that we trust it, the accuracy also depends on the shape of the axis and r/r_crit etc. In all cases we know the axis is "most accurate" though the exact degree of falloff isn't clear

Reply:
I agree, the order of the NAE would not relate to the weighting near-axis that we are trying to do here.